app.js

```diff
@@ -1,44 +1,63 @@
 const parse = require('csv-parser')
 const fs = require('fs')
 
-const fecha1 = new Date(2021,1,28)
-const fecha2 = new Date(fecha1.getFullYear(), fecha1.getMonth(), fecha1.getDate()+15)
+const fechaInicio = process.argv[2]
+const fechaFin = process.argv[3]
+
+const fecha1 = new Date(fechaInicio)
+const fecha2 = new Date(fechaFin)
 
 const cumpleanios = []
 
-// Enumero a los cumpleañeros y muestro solo su fecha de nacimiento y su nombre
-const mostrar = (datos) =>{
+function isValidDate(date) {
+    let regex = new RegExp(/^([12]\d{3}\/(0[1-9]|1[0-2])\/(0[1-9]|[12]\d|3[01]))$/)
+    let dateOk = regex.test(date)
+    if (dateOk) {
```

Comment:
```suggestion
    return dateOk
```

Reply:
Acá la idea es sacar el if (agrega complejidad ciclomática y la variable dateOk es suficiente para resolver).

Otra opción, pero es menos verbosa

```suggestion
   return regex.text(date)
```

O hacerlo en una función de una sola linea tipo

```suggestion
const isValidDate = (date) => ....
```